test/packaging/ansible/installation-root.yml

```diff
@@ -4,10 +4,10 @@
   become: true
   gather_facts: yes
 
```

Comment:
I think that this should be uncommented 👇

Reply:
done